layouts/shortcodes/read-content.html

```diff
@@ -1,4 +1,4 @@
 {{ $path := .Get "file" }}
 {{ with $.Site.GetPage $path }}
-{{ .Content | markdownify }}
+{{ .Content }}
```

Comment:
Based on my tests, `markdownify` is not necessary for properly rendering file content using this shortcode.  However, having it in here was breaking the ability for pages with the tabpane to be rendered properly (since it was somehow confusing Hugo when it tried to calculate everything....).